mountinfo/mounted_linux_test.go

```diff
@@ -385,3 +421,22 @@ func TestMountedByOpenat2VsMountinfo(t *testing.T) {
 		}
 	}
 }
+
+func TestRootIsMounted(t *testing.T) {
```

Comment:
Is there a reason that we need a separate test here? 
Can we not  make it part of `testMounts` in `TestMountedBy`?
This will help increase coverage for both stat and openat2 as well.

Reply:
Because root mount ("/") is a very separate case, it is handled by Mounted / MountedFast separately, and neither `mountedByStat` nor `mountedByOpenat2` can work with it.

Reply:
Perhaps worth adding a short GoDoc to describe that, e.g. (suggestion welcome; it's just a quick blurb)

```go
// TestRootIsMounted verifies the special handling for mounting root ("/")
```

I'd be in favor of naming the test `TestMountedXXXX`; it helps filtering tests

Reply:
done